bundles/org.eclipse.equinox.console.ssh/META-INF/MANIFEST.MF

```diff
@@ -13,8 +13,6 @@ Import-Package: javax.security.auth;resolution:=optional,
  javax.security.auth.login;resolution:=optional,
  javax.security.auth.spi;resolution:=optional,
  org.apache.felix.service.command;version="[1.0,2.0)",
- org.apache.sshd.common;version="2.4.0";resolution:=optional,
```

Comment:
This has to be investigated, as it leads to

> Access restriction: The type 'RuntimeSshException' is not API

@tjwatson what I specifically wonder is, is the sshd really _optional_? As only a test complains do we have the case that sshd is actually only used by test code?

Reply:
I believe console.ssh implementation without actual ssh makes no sense thus I'll drop the optional.